.github/workflows/build.yml

```diff
@@ -59,3 +59,6 @@ jobs:
         with:
           name: logs_${{ github.run_id }}
           path: build/logs/
+
+      - name: Push to GitHub Packages
+        run: dotnet nuget push "build/artifacts/**/*.nupkg" --source https://nuget.pkg.github.com/wixtoolset/index.json --api-key ${{ secrets.GITHUB_TOKEN }} --skip-duplicate
```

Comment:
This probably needs to be restricted to not run in PR builds and not run in forked repos.

Reply:
Yeah, that's the last step. It is very possible this go around will fail because one or more .nupkgs will need their `RepositoryUrl` set properly.

Reply:
Is this resolved? What's going to happen when I sync my repo and the build automatically kicks off? My build fails because it doesn't have rights to push to the `wixtoolset` feed?

Reply:
I did take the repository_owner out of the URL so I now added it to the condition.